### ISSUE TITLE
Uncaught TypeError when removing links in Obsidian 1.4.13

### ISSUE THREAD

**Issue by fanis** [Benature/obsidian-text-format#64]
Using the latest version of the plugin 2.2.7 on latest Obsidian 1.4.13.

Other features like "Title Case" or "Uppercase" work, but not "Remove Wikilinks" or "Remove URL Links".

Tested on the following new note:
```
## heading

this is a link to [[#heading|my heading]] and some Other Text
```

I select the line that includes the link and CTRL+P select "Remove Wikilinks in Selection". 
I expect the link to be removed according to my preference (keep `{text}`).
What happens instead is the link remains and 1 new line is added to the top of the note, which is focused.

In Developer Console I see:

```
plugin:obsidian-text-format:483 
Uncaught TypeError: Cannot read properties of null (reading 'groups')
    at eval (plugin:obsidian-text-format:483:27)
    at String.replace (<anonymous>)
    at removeWikiLink (plugin:obsidian-text-format:480:14)
    at TextFormat.textFormat (plugin:obsidian-text-format:1192:32)
    at Object.callback (plugin:obsidian-text-format:817:38)
    at o_ (app.js:1:1834749)
    at t.onChooseItem (app.js:1:2390850)
    at t.onChooseSuggestion (app.js:1:1936680)
    at t.selectSuggestion (app.js:1:1936230)
    at e.useSelectedItem (app.js:1:1451982)
```

**Comment by Benature** [comment]
Sorry that I missed considering about the case without filename, it should be fixed in v2.2.8. Please have a try. :)

**Comment by fanis** [comment]
Thank you, works well now.
